### DISCUSSION TITLE
小狼毫支持万能键吗？

### DISCUSSION THREAD

**Discussion by GitFlzy** [rime/home#1690] [Q&A] [answered]
我现在的输入方案基于小鹤音形，全码是四码，前两码是拼音，后两码是字形，现在我想要一个万能键，小狼毫支持这种功能吗？

这个所谓万能键的基本逻辑是，一个万能键 ` 作为一个能代替任意字母的占位，候选字/词/句就只显示当前层级的编码（比如说 话是 hx，画 是 hxa，输入 h` 不会候选到 hxa），因为我的方案基本是 4 码，且它算是一个形码，编码不会太多，应该没有什么性能问题，我设想就是一个字典查找树，但不会在小狼毫上弄。

候选表里显示后续可能的编码不能替代万能键，因为我可能遗忘或者根本不知道中间部分的编码；启用其它编码作反查看起来也不是很好，首先我很可能要自己重新弄，其次是小鹤音形本来就是音+形，绝大部分情况下不会两部分都忘了，而我这里的编码量少，万能键翻一翻很快能翻到。

**Comment by yrljroli** [comment, marked as answer]
参#1300。

> 首先我很可能要自己重新弄，

这个工作有人作过了。<https://github.com/kchen0x/rime-crane/blob/main/xhup_reverse.dict.yaml>。
